lua/lazy/core/config.lua

```diff
@@ -31,6 +31,20 @@ M.defaults = {
     -- increase downloads a lot.
     filter = true,
   },
+  pkg = {
+    enabled = true,
+    cache = vim.fn.stdpath("state") .. "/lazy/pkg-cache.lua",
+    versions = true, -- Honor versions in pkg sources
+    sources = {
```

Comment:
as far as I can tell by looking at the code this config option determines also priority
as soon as a source is found the other ones won't be looked at
this is quite important in my opinion and should be documented

Reply:
A new doc website is in the works https://lazy.folke.io

Still need to rewrite a lot of stuff and document packages

Reply:
New docs have been updated https://lazy.folke.io/